src/lib/Scenes/Map/Components/CitySwitcherButton.tsx

```diff
@@ -72,6 +72,6 @@ export class CitySwitcherButton extends Component<Props> {
           )}
         </Background>
       </TouchableWithoutFeedback>
-    )
+    ) : null
```

Comment:
Aside: This could be a 2nd `||` instead of the ternary, false won't render in a JSX tree

Reply:
Ah yeah, that makes sense. So you could get a kind of null-coalescing component tree, like `??` in Swift.